src/panels/lovelace/entity-rows/hui-toggle-entity-row.ts

```diff
@@ -0,0 +1,68 @@
+import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
+import { TemplateResult } from "lit-html";
+
+import "../components/hui-generic-entity-row.js";
+import "../../../components/entity/ha-entity-toggle.js";
+
+import computeStateDisplay from "../../../common/entity/compute_state_display.js";
+import { hassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
+import { HomeAssistant } from "../../../types.js";
+import { EntityRow, EntityConfig } from "./types.js";
+import { HassEntity } from "home-assistant-js-websocket";
+
+class HuiToggleEntityRow extends hassLocalizeLitMixin(LitElement)
+  implements EntityRow {
+  public hass?: HomeAssistant;
+  private _config?: EntityConfig;
+
+  static get properties(): PropertyDeclarations {
+    return {
+      hass: {},
+      _config: {},
+    };
+  }
+
+  public setConfig(config: EntityConfig): void {
+    if (!config || !config.entity) {
+      throw new Error("Invalid Configuration: 'entity' required");
+    }
+
+    this._config = config;
+  }
+
+  protected render(): TemplateResult {
+    if (!this._config || !this.hass || !this.hass.states[this._config.entity]) {

```

Comment:
This is not good. Right now we won't render anything if the entity doesn't exist.

The old behavior was that if the entity doesn't exist, we still render `<hui-generic-entity-row>` which would then go ahead and render a yellow warning that the entity didn't exist.